cmobase/Zicboz.adoc

```diff
@@ -5,6 +5,10 @@ Cache block zero instructions perform a series of byte-sized store operations
 where the data are zero. An implementation may or may not update the entire
 cache block atomically.
 
+The cache block zero instructions can operate even if the target memory
```

Comment:
I'm tempted to make this a non-normative comment, since the architecture does not make this distinction for normal stores.

Reply:
My  concern here from the software point of view is that we'd like this instruction to behave consistently across all implementations.  Most of the language in these specifications refers to cache blocks.  If an implementor takes the "cache block" aspect too literally, it would be unfortunate if CBO.ZERO did not succeed against uncacheable or non-cached address ranges on some platforms

Reply:
Oh. I agree we need to tighten up the definition of what CBO.ZERO works on, which is less a cache block and more the NAPOT region corresponding to a cache block. I will definitely update that, but given that new definition, I don't think we need to tread into the differences between cacheable and non-cacheable attributes. Would you agree?

Reply:
Yes - agreed